dependency-check-suppressions.xml

```diff
@@ -17,12 +17,11 @@
 	</suppress>
 
 	<suppress>
-		<cve>CVE-2020-8908</cve>
+		<notes>Analysis: No fix or work around available</notes>
```

Comment:
There are 2 blocks of 'Analysis: No fix or work around available'. Could you merge them into 1?